x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_about_rule/default_value.ts

```diff
@@ -33,5 +33,6 @@ export const stepAboutDefaultValue: AboutStepRule = {
   timestampOverride: '',
   threat: threatDefault,
   note: '',
+  maxSignals: 100,
```

Comment:
Does it make sense to import this value from [constants](https://github.com/dplumlee/kibana/blob/1cb02a24b0f72b32114b04e3564ee00207e776f4/x-pack/plugins/security_solution/common/constants.ts#L50)?

I noticed that `DEFAULT_MAX_SIGNALS` is used on the backend when calculating the final max signals value [here](https://github.com/dplumlee/kibana/blob/1cb02a24b0f72b32114b04e3564ee00207e776f4/x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts#L321).